examples/calibration/plot_compare_calibration.py

```diff
@@ -150,48 +156,84 @@ def predict_proba(self, X):
 plt.show()
 
 # %%
-# :class:`~sklearn.linear_model.LogisticRegression` returns well calibrated
-# predictions as it directly optimizes log-loss. In contrast, the other methods
-# return biased probabilities, with different biases for each method:
 #
-# * :class:`~sklearn.naive_bayes.GaussianNB` tends to push
-#   probabilities to 0 or 1 (see histogram). This is mainly
-#   because the naive Bayes equation only provides correct estimate of
-#   probabilities when the assumption that features are conditionally
-#   independent holds [2]_. However, features tend to be positively correlated
-#   and is the case with this dataset, which contains 2 features
-#   generated as random linear combinations of the informative features. These
-#   correlated features are effectively being 'counted twice', resulting in
-#   pushing the predicted probabilities towards 0 and 1 [3]_.
+# :class:`~sklearn.linear_model.LogisticRegressionCV` returns reasonably well
+# calibrated predictions despite the small training set size: its reliability
+# curve is the closest to the diagonal. The reason is two-fold: this estimator
+# directly adjusts its trainable parameter to minimize negative loglikelihood
+# (a.k.a. "log-loss") on a given training set. Furthermore we configured it to
+# automatically tune the `C` regularization parameter to minimize the negative
+# loglikelihood via inner cross-validation.
+#
+# The negative loglikelihood is a proper scoring rule for probabilistic
+# classification models. It means, that in the limit of infinite training data,
+# it is minimized by a model that predicts the true conditional probabilities
```

Comment:
From here `it` is used a few times and can be hard to see that `it== negative loglikelihood

```suggestion
# classification models. In the limit of infinite training data, the
# negative log-likelihood is minimized by a model that predicts the true conditional probabilities
```